packages/testing-utils/types/index.d.ts

```diff
@@ -107,7 +107,8 @@ export interface SagaIteratorClone extends SagaIterator {
 export function createMockTask(): MockTask
 
 export interface MockTask extends Task {
-  setRunning(running: boolean): void
+  setRunning(): void
```

Comment:
We could add a `@deprecated` signature back and handle both at the implementation level - `task.setRunning()` should just be synonymous with `task.setRunning(true)`. I'm not 100% sure what `task.setRunning(false)` should correspond to though... it seems like canceled would fit better here than aborted

Reply:
For all of the times I've used this in the past, I've always had to do:

```
mockTask.setResult(X)
mockTask.setRunning(false)
```

This is necessary to get the rest of the saga to correctly proceed when joining on a mock task.

I've never had to do `setRunning(true)` because the tasks are already running when created.

I think taking out setRunning altogether is simpler than trying to guess what people would expect. Maybe that means publishing another major version? I think since it's a testing utility if we did it in a minor version it would be acceptable.